robottelo/cli/contentview.py

```diff
@@ -124,14 +124,18 @@ def publish(cls, options, timeout=1500):
         )
 
     @classmethod
-    def version_info(cls, options):
+    def version_info(cls, options, output_format=None):
         """Provides version info related to content-view's version."""
         cls.command_sub = 'version info'
 
         if options is None:
             options = {}
 
-        return hammer.parse_info(cls.execute(cls._construct_command(options)))
+        result = cls.execute(
+            cls._construct_command(options), output_format=output_format)
+        if output_format != 'json':
+            result = hammer.parse_info(result)
```

Comment:
CVV with puppet modules has multiline description, which is currently not parsed correctly (see #5681). Adding support for different output formats, we are supposed to move to `json` output regardless.